cfgov/jinja2/v1/_includes/organisms/well.html

```diff
@@ -14,10 +14,6 @@
 
    ========================================================================== #}
 
-{% if not value %}
-    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
-    {% do global_dict.update({'well': { 'content' :  lipsumText  }}) %}
-{% endif %}
 {% set value = value or global_dict.well %}
```

Comment:
This should be removed too. Any instance of this should be removed in the codebase for the time being.